lib/WebpackConfig.js

```diff
@@ -51,6 +51,21 @@ class WebpackConfig {
         this.babelConfigurationCallback = function() {};
         this.useReact = false;
         this.loaders = [];
+        this.useTypeScriptLoader = false;
+        this.typeScriptOptions = {
+            transpileOnly: false,
+            happyPackMode: false,
+            logInfoToStdOut: false,
+            logLevel: 'info',
+            silent: false,
+            ignoreDiagnostics: [],
+            compiler: 'typescript',
+            configFileName: 'tsconfig.json',
+            visualStudioErrorFormat: false,
+            compilerOptions: {},
+            entryFileIsJs: false,
+            appendTsSuffixTo: []
+        };
```

Comment:
These are the same default values taken from `ts-loader` I add them just to follow the same scheme of implementation Encore had, but if that validation will be removed we can remove also those defaults. 

Also if configuration by callback is added then makes no sense since we are delegating all of that to users. Make sense to make other validations for sanity check but not these currently.